### ISSUE TITLE
Correct workflow to merge multiple PRs

### ISSUE THREAD

**Issue by focusaurus** [greenkeeperio/greenkeeper#126]
So I'd like to be able to ignore greenkeeper PRs, which are quite frequent, until I have a moment to process them all in a batch. I tried merging the oldest first which seemed logical to me, but that automatically closes all the newer PRs even for unrelated packages. Am I just doing it wrong here?

**Comment by boennemann** [comment]
Hey @focusaurus,

currently merging multiple PRs at the same time is a bit tedious, as it requires manual work.
We're trying to roll out https://github.com/greenkeeperio/greenkeeper/issues/137 as soon as possible to address this.

Merging one Greenkeeper PR should not auto-close PRs of unrelated dependencies. Full details [here](https://github.com/greenkeeperio/greenkeeper/issues/62#issuecomment-176248343).

> Whenever you **merge a Greenkeeper pull request** we **will delete**:
- the branch of that PR
- all branches (hence automatically closing the PRs) that are
  1. opened by Greenkeeper
  2. for the same dependency
  3. versions:
    1. lower as the one being merged, or
    2. matching the same version range


If this happened nonetheless please point us to the repo, PRs and branches where this happened so we can make sure it never happens again.

Thanks so much for your feedback.

Best,
Stephan

**Comment by focusaurus** [comment]
OK so I suspect this was at least partially some confusion on my part. I think one of my scripts when I pushed from my local machine to my personal git server, which then mirrors to github, was using the `--mirror` command line flag which blasted away the greenkeeper PRs. I probably misattributed this behavior to greenkeeper when it was in fact that post-receive script.

Once I corrected that to `--all` I think the problem went away.

I can now usually do something like this:

- scan list of all open PRs
- bulk select any not-the-latest-version-PRs and close them
- proceed through the remaining reverse chronologically and merge and delete (I think this usually works without conflicts but haven't had enough instances to say for sure)

That sort of works.

However, FYI it's a pain so I've decided in cases where there's more than a few backlogged PRs, to just use command line `npm-check-updates` to handle it and bulk close all the greenkeeper ones.
